docker/storage/cassandra/Dockerfile

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright 2015-2019 The OpenZipkin Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+#
+
+# Share the same base image to reduce layers used in testing
+FROM openzipkin/jre-full:1.8.0_212
+MAINTAINER OpenZipkin "http://zipkin.io/"
+
+ENV ZIPKIN_VERSION 2.18.0
+
+ENV CASSANDRA_VERSION=3.11.4 \
+    JAVA=/usr/local/java/jre/bin/java \
+    JVM_OPTS="-Dcassandra -Djava.net.preferIPv4Stack=true"
+
+WORKDIR /cassandra
+
+ADD zipkin-storage/cassandra/src/main/resources/*.cql /zipkin-schemas/
```

Comment:
easier! soon ideally we can generate an elasticsearch one using the main class, also cc @basvanbeek